Makefile

```diff
@@ -0,0 +1,86 @@
+.DEFAULT_GOAL := spec
+
+
+# Builds for the current platform
+build:
+	go install
+
+# Makes a new binary release
+build-release: cross-compile
+	package/debian/make_deb.sh
+
+# Builds the binary for all platforms
+cross-compile:
+	go get github.com/mitchellh/gox
+	timestamp=$(TZ=UTC date -u '+%Y-%m-%dT%H:%M:%SZ')
+	sha=$(git rev-parse HEAD)
+	gox -ldflags "-X github.com/Originate/git-town/cmd.Version=$TRAVIS_TAG -X github.com/Originate/git-town/cmd.BuildTime=$timestamp) -X github.com/Originate/git-town/cmd.GitHash=$sha" \
+			-output "dist/{{.Dir}}-{{.OS}}-{{.Arch}}"
+
+# Runs the feature tests
+cuke: build
+	bundle exec parallel_cucumber $(DIR)
+DIR = $(if $(dir),$(dir),"features")
+
+# Deploys the website
+deploy:
+	git checkout gh-pages
+	git pull
+	git checkout master
+	git pull --rebase
+	harp compile website/ _www
+	git checkout gh-pages
+	cp -r _www/* .
+	rm -rf _www
+	git add -A
+	git commit
+	git push
+	git checkout master
+
+# Fixes all issues in all languages
+fix: fix-cucumber fix-ruby fix-markdown
+
+# Fixes all Cucumber issues
+fix-cucumber:
+	bundle exec cucumber_lint --fix
+
+# Fixes all Markdown issues
+fix-markdown:
+	prettier --write "{,!(vendor)/**/}*.md"
+
+# Fixes all Ruby issues
+fix-ruby:
+	bundle exec rubocop --auto-correct
+
+# Lints all the source code
+lint: lint-go lint-markdown lint-ruby
+
+lint-go:
+	goimports -d src
+	gometalinter.v2
```

Comment:
@charlierudolph I am upgrading to v2 here, and no longer run gometalinter in debug mode. This will be hard to make work in a Makefile, and it was very hacky anyways. We should only use the normal output of gometalinter in my opinion. It finds enough things.